index.ejs

```diff
@@ -1,26 +1,21 @@
 <!DOCTYPE html>
 <html>
-
 <head>
   <title>Tidepool</title>
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
   <meta name="viewport" content="width=device-width,initial-scale=1.0">
-  <!-- Crowdin Start -->
-  <!-- Crowdin End -->
 </head>
-
 <body>
   <div id="app">
     <p class="loading">Loading app...</p>
   </div>
   <!-- config -->
   <!-- Start of support Zendesk Widget script -->
-  <script id="ze-snippet" type="text/javascript" src="https://static.zdassets.com/ekr/snippet.js?key=d1aad8b3-67d8-4f44-9d87-9e8a0ed8149d"></script>
   <!-- End of support Zendesk Widget script -->
   <!-- Start of Tracker Code -->
-  <script type="text/javascript" src="matomo.js"></script>
   <!-- End of Tracker Code -->
+  <!-- Start of Crowdin -->
+  <!-- End of Crowdin -->
```

Comment:
According to Crowdin doc:
It's important to add the Crowdin In-Context script before any other scripts, at the <head> section of your document. Make sure this script is included on every page with localizable text. Optionally, you can include this JavaScript snippet only when the pseudo-language is active.